yt_dlp/extractor/rtvslo.py

```diff
@@ -7,14 +7,17 @@
     unified_timestamp,
     url_or_none,
 )
+import re
+from urllib.parse import urlparse
 
 
 class RTVSLOIE(InfoExtractor):
     IE_NAME = 'rtvslo.si'
     _VALID_URL = r'''(?x)
         https?://(?:
             (?:365|4d)\.rtvslo.si/arhiv/[^/?#&;]+|
-            (?:www\.)?rtvslo\.si/rtv365/arhiv
+            (?:www\.)?rtvslo\.si/rtv365/arhiv|
+            (?:365|4d)\.rtvslo.si/oddaja/[^/?#&;]+|
```

Comment:
I would move this to a separate extractor instead of having a single extractor deal with both individual videos and playlists of videos.